task/tests/test.sh

```
@@ -0,0 +1,33 @@
+# separate verifier image. pytest is baked into environment/Dockerfile, so do NOT install or
+# download anything here — verify-time setup is rejected by the static checks.
+#
+# Put your pytest files (e.g. test_outputs.py) in tests/ and run them below. Harbor overlays
+# tests/ at /tests only at verify time, so keep ground truth / expected outputs in tests/
+# (never in environment/, where the agent could read them).
+# --ctrf writes a standard JSON report; write 1/0 to /logs/verifier/reward.txt.
+#!/bin/bash
+#
+# 1. Create the test data
+echo "date,product,quantity,price" > /workspace/sales.csv
+echo "2024-01-01,Widget,10,5.0" >> /workspace/sales.csv
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)